scripts/web-dev-deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+BUILD_JAR=$(ls /home/ec2-user/hellogsm-server/hellogsm-web/build/libs/*.jar)
+JAR_NAME=$(basename $BUILD_JAR)
+
+DEPLOY_PATH=/home/ec2-user/dev/web/sictak/
+cp $BUILD_JAR $DEPLOY_PATH
+
+CURRENT_PID=$(pgrep -f $JAR_NAME)
+
+if [ -z $CURRENT_PID ]
+then
+  echo "현재 구동중인 애플리케이션이 없으므로 종료하지 않습니다."
+else
+  echo "> kill -15 $CURRENT_PID"
+  kill -15 $CURRENT_PID
+  sleep 5
+fi
+
+DEPLOY_JAR=$DEPLOY_PATH$JAR_NAME
+chmod +x $DEPLOY_JAR
+nohup java -jar &
```

Comment:
해당 부분이 ec2의 jar파일을 실행시키는 부분이라면, 실행 시 spring profile 설정을 `dev`로 해 줘야 합니다.
```suggestion
nohup java -jar $DEPLOY_JAR --spring.profiles.active=dev &
```
이런식으로 변경하면 동작할 것 같은데, 확실하진 않네요.

Reply:
바꾸고 동작 잘 됩니다 감사합니다!